packages/trader/src/Services/ws-methods.js

```diff
@@ -111,6 +119,7 @@ const WS = (() => {
         landingCompany,
         logout,
         mt5LoginList,
+        newVirtualAccount,
```

Comment:
do we want this format instead? `newAccountVirtual` so in future we use `newAccountReal` etc?